src/neuroconv/tools/testing/converter_pipe_mixins.py

```diff
@@ -0,0 +1,92 @@
+import json
+import tempfile
+from abc import abstractmethod
+from pathlib import Path
+from typing import List, Type, Union
+
+from jsonschema.validators import Draft7Validator, validate
+
+from neuroconv import ConverterPipe
+from neuroconv.utils import NWBMetaDataEncoder
+
+
+class ConverterPipeTestMixin:
```

Comment:
This should be done separately, first is sufficient to add the tests specific for this converter and can be revised after.